### PR TITLE
Add resolution support CLI

### PR DESCRIPTION
**Pull request by pvizeli**

### CODE REVIEW

File: cmd/resolution_suggestion.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var resolutionSuggestionCmd = &cobra.Command{
+	Use:     "suggestion",
+	Aliases: []string{"su", "solve"},
```
Comment (mjrider) [inline cmd/resolution_suggestion.go:9]: solve is a bit of a strange alias for suggestion

File: cmd/resolution.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	Use:     "resolution",
+	Aliases: []string{"resolutions", "res"},
+	Short:   "Resolution center of Supervisor, show issues and suggest solutions",
+	Long: `
+The Resolution center provide information about detected issue on the system. It allow also 
+to give suggestion which can fix the issue and can executed by the suggestion command. Othewise
+it's possible to dismiss issues or suggestion. It show also why a system show as not supported.`,
+	Example: `
+  ha resolution info
+  ha resolution suggestion apply [ID]`,
```
Comment (ludeeus) [inline cmd/resolution.go:17]:
```suggestion
  ha resolution suggestion apply [id]`,
```
Looks like all others are lower case

File: cmd/resolution_issue_dismiss.go [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var resolutionIssueDismissCmd = &cobra.Command{
+	Use:     "dismiss",
+	Aliases: []string{"disable", "remove"},
+	Short:   "Issues dismiss reported by Resolution center",
+	Long: `
+This command allow to dismiss issues reported by the System.`,
```
Comment (ludeeus) [inline cmd/resolution_issue_dismiss.go:20]:
```suggestion
This command allows dismissing issues reported by the system.`,
```

File: cmd/resolution_issue_dismiss.go [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+	helper "github.com/home-assistant/cli/client"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var resolutionIssueDismissCmd = &cobra.Command{
+	Use:     "dismiss",
+	Aliases: []string{"disable", "remove"},
+	Short:   "Issues dismiss reported by Resolution center",
```
Comment (ludeeus) [inline cmd/resolution_issue_dismiss.go:18]:
```suggestion
	Short:   "Dismiss issues",
```

File: cmd/resolution_suggestion.go [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+import (
+	"github.com/spf13/cobra"
+)
+
+var resolutionSuggestionCmd = &cobra.Command{
+	Use:     "suggestion",
+	Aliases: []string{"su", "solution"},
+	Short:   "Suggestion management reported by Resolution center",
+	Long: `
+This command allow to dismiss or apply suggestion reported by the System.`,
```
Comment (ludeeus) [inline cmd/resolution_suggestion.go:12]:
```suggestion
This command allows to dismiss or apply suggestion reported by the system.`,
```

File: cmd/resolution_suggestion_apply.go [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+)
+
+var resolutionSuggestionApplyCmd = &cobra.Command{
+	Use:     "apply",
+	Aliases: []string{"enable", "run"},
+	Short:   "Suggestion apply reported by Resolution center",
+	Long: `
+This command allow to apply an suggestion reported by the System.`,
+	Example: `
+  ha resolution suggestion dismiss [id]`,
```
Comment (ludeeus) [inline cmd/resolution_suggestion_apply.go:22]:
```suggestion
  ha resolution suggestion apply [id]`,
```

File: cmd/resolution_suggestion_dismiss.go [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var resolutionSuggestionDismissCmd = &cobra.Command{
+	Use:     "dismiss",
+	Aliases: []string{"disable", "remove"},
+	Short:   "Suggestion dismiss reported by Resolution center",
+	Long: `
+This command allow to dismiss a suggestion reported by the System.`,
```
Comment (ludeeus) [inline cmd/resolution_suggestion_dismiss.go:20]:
```suggestion
This command allows dismissing a suggestion reported by the system.`,
```

**Review by ludeeus — APPROVED** [review]
LGTM :+1:

**Pull request merged**